numpy/core/_add_newdocs.py

```diff
@@ -8009,22 +8022,65 @@ def luf(lamdaexpr, *args, **kwargs):
 
 add_newdoc('numpy.core.numerictypes', 'float128',
     """
-    128-bit floating-point number. Character code: 'g'. C long float
+    128-bit floating-point number. Character code: ``'g'``. C long float
     compatible.
 
     """)
 
 add_newdoc('numpy.core.numerictypes', 'int8',
-    """8-bit integer. Character code ``b``. C char compatible.""")
+    """
+    8-bit integer. Character code ``'b'``. C char compatible.
+
+    """)
 
 add_newdoc('numpy.core.numerictypes', 'int16',
-    """16-bit integer. Character code ``h``. C short compatible.""")
+    """
+    16-bit integer. Character code ``'h'``. C short compatible.
+
+    """)
 
 add_newdoc('numpy.core.numerictypes', 'int32',
-    """32-bit integer. Character code 'i'. C int compatible.""")
+    """
+    32-bit integer. Character code ``'i'``. C int compatible.
+
+    """)
 
 add_newdoc('numpy.core.numerictypes', 'int64',
-    """64-bit integer. Character code 'l'. Python int compatible.""")
+    """
+    64-bit integer. Character code ``'l'``. Python int compatible.
```

Comment:
Wondering a bit about the Python int compatible part here. It is mostly valid for python2, and even then wrong on windows (and 32bit linux).

Reply:
Yeah, this is a) not true on 32-bit windows, and b) doesn't set `__doc__` correctly anyway

Reply:
`np.dtype('int64').__doc__` is empty, but `np.int64.__doc__` shows this string.

Reply:
My mistake, I was misinterpreting the results of #10106